src/clj/puppetlabs/services/legacy_routes/legacy_routes_service.clj

```diff
@@ -21,7 +21,14 @@
           puppet-version (get-in config [:puppet-server :puppet-version])
           ca-settings (ca/config->ca-settings (get-config))
           ca-mount (get-route (tk-services/get-service this :CaService))
-          master-mount (master-service/get-master-path config)
+          master-ns (keyword (tk-services/service-symbol
+                               (tk-services/get-service this :MasterService)))
```

Comment:
Do you intend to also change the PE Master Service to implement the `MasterService` protocol?  Currently, it implements `PEMasterService`, which means this won't work.  (hope I have that all sorted correctly ... could be that I'm missing something here.)